test/runnable/cstuff2.c

```diff
@@ -188,6 +188,15 @@ void test11()
 
 /*********************************/
 
+void test12()
+{
+    int i;
+    i = (int) { 3 };
+    if (i != 3) { printf("test12\n"); exit(1); }
```

Comment:
This should be more thoroughly tested:

- empty type
- empty initializer compound statement
- more than 1 expressions inside the compound statement
- the hdrgen code should also be tested

Reply:
There are still many shortcomings in the initializer semantics that I need to address. But that isn't what this PR is about.